src/main/java/org/terasology/navgraph/BitMap.java

```diff
@@ -114,6 +141,11 @@ public String toString() {
         return sb.toString();
     }
 
+    /**
+     *
+     * @return Number of Cells in the map
```

Comment:
First letter should be lowercase here, too (anywhere after `@return` and `@param`, per https://github.com/MovingBlocks/Terasology/wiki/Javadoc - I won't mark the rest of them)